modules/imgproc/include/opencv2/imgproc/imgproc_c.h

```diff
@@ -396,7 +396,7 @@ CVAPI(void)  cvGetRectSubPix( const CvArr* src, CvArr* dst, CvPoint2D32f center
     matrixarr = ( a11  a12 | b1 )   dst(x,y) <- src(A[x y]' + b)
                 ( a21  a22 | b2 )   (bilinear interpolation is used to retrieve pixels
                                      with fractional coordinates)
-@see cvWarpAffine
+@see cv::getQuadrangleSubPix
```

Comment:
There is no `cv::getQuadrangleSubPix` function declared in OpenCV so it's a bit confusing to reference it in documentation

Reply:
Thanks @terfendail . Actually I am referencing about this **"CVAPI(void)  cvGetQuadrangleSubPix"** function as in the module in the starting of every function there is declaration like **@see "function name"** which here is different as the function proceeding is **"CVAPI(void)  cvGetQuadrangleSubPix"** but here written is **cvWarpAffine** . Therefore , I have done it.  Please, suggest ..